src/ucp/api/ucp_def.h

```diff
@@ -101,7 +101,10 @@ typedef enum {
     UCP_ERR_HANDLING_MODE_NONE,             /**< No guarantees about error
                                              *   reporting, imposes minimal
                                              *   overhead from a performance
-                                             *   perspective */
+                                             *   perspective. @note
+                                             *   @ref ucp_ep_params_t::err_handler
+                                             *   is not used for errors
+                                             *   reporting in this mode. */
```

Comment:
Grammar and phrasing fix: "@note In this mode, @ref ucp_ep_params_t::err_handler is not used for error reporting."